espnet2/diar/loss/wrappers/flexible_spk_pit_solver.py

```diff
@@ -0,0 +1,68 @@
+from itertools import permutations
```

Comment:
This file has a lot of overlap as espnet2/enh/loss/wrapper/pit_solver.py. Probably you can combine those two? The difference is only that num_spk determined by inf or ref. Maybe set an option to choose?

Reply:
Added an option to pit_solver.py